src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/components/CityZoneMessage.tsx

```diff
@@ -144,6 +136,72 @@ const CityZoneBox = ({
   );
 };
 
+const useActionButtons = (cityZone?: CityZone) => {
+  const {t, language} = useTranslation();
+  const analytics = useAnalytics();
+
+  if (!cityZone) {
+    return [];
+  }
+
+  const actionButtons: ActionButton[] = [];
+  const orderUrl = getTextForLanguage(cityZone.orderUrl, language);
+  if (orderUrl) {
+    actionButtons.push({
+      id: `book_online_action`,
+      text: t(CityBoxMessageTexts.actionButtons.bookOnline),
+      icon: ExternalLink,
+      interactiveColor: 'interactive_0',
+      accessibilityHint: t(CityBoxMessageTexts.a11yHintForExternalContent),
+      onPress: () => {
+        analytics.logEvent('Flexible transport', 'Book online url opened', {
+          name: 'book_online_action',
```

Comment:
It's not necessary to add this `name` property. The event name will be `Flexible transport: Book online url opened`, and that's descriptive enough. You can remove name from the events below as well.